pyvista/examples/downloads.py

```diff
@@ -3364,6 +3364,94 @@ def download_action_figure(load=True):  # pragma: no cover
     return _download_and_read('tigerfighter.obj', load=load)
 
 
+def download_sun_jpg():  # pragma: no cover
```

Comment:
I would prefer keeping with our existing API of always returning a PyVista object, but including the optional `load=True` parameter.